pkg/middleware/util.go

```diff
@@ -4,11 +4,14 @@ import (
 	"strings"
 
 	"github.com/go-macaron/gzip"
+	"github.com/grafana/grafana/pkg/infra/log"
 	"gopkg.in/macaron.v1"
 )
 
+var gziperLogger log.Logger = log.New("gziper")
```

Comment:
I would put this into `Gziper()` function and assign variables similar to `gziper := gzip.Gziper()` creating a variable locally scoped to the function instead of package scoped.

Reply:
Good point @marefr.